pyinfra/modules/virtualenv.py

```diff
@@ -0,0 +1,44 @@
+# pyinfra
+# File: pyinfra/modules/pip.py
+# Desc: manage virtualenvs
+
+'''
+Manage Python virtual environments
+'''
+
+from __future__ import unicode_literals
+
+from pyinfra.api import operation
+from pyinfra.modules import files
+
+
+@operation
+def virtualenv(
+    state, host,
+    path, python='python3', site_packages=False, always_copy=False,
```

Comment:
Could we default python to `None` and when present add it to the args, otherwise defaulting to the system Python version (eg. not having to specify `python='python2'` on Ubuntu 14, for example).